suite-native/navigation/src/routes.ts

```diff
@@ -95,6 +96,7 @@ export enum WipeDeviceStackRoutes {
     WipeDevice = 'WipeDevice',
     ContinueOnTrezor = 'ContinueOnTrezor',
     WipeDeviceLoadingScreen = 'WipeDeviceLoadingScreen',
+    FactoryReset = 'FactoryReset',
```

Comment:
Close button inside app is not working at all for me during Factory reset (Clicking on X in app does nothing). Canceling the Factory reset on T1 get's me to inconsistent state because T1 want's to reconnect and app looks like it can re-run it.


https://github.com/user-attachments/assets/12c3904e-ecb9-4a2c-882e-dfd52f9f9bbc



Device state after cancelation on the trezor:
<img width="330" alt="image" src="https://github.com/user-attachments/assets/7aca5ffb-2cb6-486d-b21a-ea1e76bb21a0" />

Reply:
As discussed, only the close button was hidden. It's the same on desktop, once factory reset starts, user is not able to cancel it without unplugging device. Hardware back button will not do anything either.
https://github.com/trezor/trezor-suite/pull/19433/commits/d67a26c3395de018140cf27418038a6918a41fc1